mozilla-release/toolkit/mozapps/extensions/internal/XPIProvider.jsm

```diff
@@ -7943,7 +7943,9 @@ Object.assign(SystemAddonInstallLocation.prototype, {
       logger.warn(`System add-on ${aAddon.id} isn't compatible with the application.`);
       return false;
     }
-
+    /**
+     * CLIQZ - https-everywhere is unpacked non bootstrap addon, though we want
+     *         it to be a system addon. Thus we disable those checks for now
```

Comment:
This is usually a doc-comment format. Looks alien here.
And please remove the code with preprocessor directive instead of a comment.